providers/hubspot/errors.go

```diff
@@ -52,11 +52,7 @@ type ErrLinks struct {
 
 func (c *Connector) interpretError(res *http.Response, body []byte) error {
 	mediaType, _, err := mime.ParseMediaType(res.Header.Get("Content-Type"))
-	if err != nil {
```

Comment:
Similar to issue handled in this PR https://github.com/amp-labs/connectors/pull/1085

Made change for hubspot as well.